### PR TITLE
Ge4 1724

### PR DESCRIPTION
**Pull request by lazaro9318**
Update dockerfile with ubuntu generic

### CODE REVIEW

**Review by grzy512 — COMMENTED** [review]
I think there should be a develop branch not master.

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-    CONSUL_VERSION=$consul_version \
-    HASHICORP_RELEASES=$hashicorp_releases \
-    FILEBEAT_VERSION=$filebeat_version \
-    CONSUL_URL=$consul_url
+ENV ZOOKEEPER_VERSION=$zookeper_version
+ENV SERVICENAME=zookeeper
+ENV CONSUL_TAGS='"web","application","prometheus"'
+ENV CONSUL_META_SCRAPE_PATH="\/metrics"
+ENV CONSUL_META_SCRAPE_PORT="7071"
+ENV LOGSTASH_VERSION=$logstash_version
```
Comment (grzy512) [inline Dockerfile]: Where LOGSTASH_VERSION env is used?

**Review by grzy512 — COMMENTED** [review]
Where zookeeper put logs ? Filebeat only takes logs from /edgeone/consul/logs/*.log and /app/logs/*.log

File: Dockerfile (hunk 1 above)
Comment (lazaro9318) [reply Dockerfile]: that was a mistake, I'm going to remove it.

**Comment by TomaszWegrzyn** [comment]
I think you need to put logs to /app/logs so they are collected by filebeat

**Comment by TomaszWegrzyn** [comment]
Oh, also plese remove consul and filebeat config files if they are no longer used

File: startup.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+chown root /edgeone/filebeat/filebeat.yml
```
Comment (lazaro9318) [inline startup.sh:3]: I added this line because it has some troubles while trying to use the file

File: Dockerfile (hunk 1 above)
Comment (lazaro9318) [reply Dockerfile]: Added env for read logs from other location

File: Dockerfile [hunk 3]
```diff
@@ -35,36 +34,26 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
...
 
+USER 1000
 
-CMD bash /usr/bin/start-zk.sh
+COPY --chown=edgeone:root startup.sh /app/startup.sh
+COPY --chown=edgeone:root service.json /app/service.json
+
+RUN chmod +x /app/startup.sh
+
+USER root
```
Comment (grzy512) [inline Dockerfile:59]: Why you switch to root here should be a edgeone user

File: startup.sh (hunk 2 above)
Comment (grzy512) [reply startup.sh:3]: This issue is related with root switch

**Pull request merged**